app/views/scheme_type.scala.html

```diff
@@ -24,43 +24,34 @@
 @ers_main(title = Messages("ers.scheme_types.page_title")) {
 
 		<p class="font-xsmall"><a href="@routes.CheckingServiceController.startPage()" class="link-back">@Html(Messages("ers.back"))</a></p>
-		@header(Messages("ers_scheme_type.page_title"))
 		@if(flash.get("scheme-not-selected-error").isDefined) {
 			@error_overview(flash.get("scheme-not-selected-error"), "error-summary")
 		}
+	<div class="@{if(flash.get("scheme-not-selected-error").isDefined) {"form-field form-field--error form-field--submit"}}">
```

Comment:
Hmm okay I don't think we need to be doing all this flashing stuff, we should be getting that out of the box. The problem comes from the fact we aren't passing the forms around properly. Argh, we might need a quick call to explain this bit.

Reply:
all flashing removed - forms being passed (again, the csv and ods uploads unchanged because it seems to be a much bigger change?)